### PR TITLE
Invert MDN logo text and icon

### PR DESCRIPTION
**Pull request by xxyzz**
Screen Shot 2022-03-03 at 20 02 52

Also invert icons like the theme toggle icon and search icon at the top right corner.

### CODE REVIEW

File: src/config/dynamic-theme-fixes.config [hunk 1]
```diff
@@ -4154,7 +4154,8 @@ CSS
 developer.mozilla.org
 
 INVERT
-.logo
+.logo_text
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:4157]:
```suggestion
.logo-text
```

File: src/config/dynamic-theme-fixes.config [hunk 2]
```diff
@@ -4154,7 +4154,8 @@ CSS
...
+.icon
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:4158]: `.icon` is too broad to be included.

File: src/config/dynamic-theme-fixes.config (hunk 1 above)
Comment (xxyzz) [reply src/config/dynamic-theme-fixes.config]: `.logo_text` actually works here but not `.logo-text`... And `span.icon` also makes the logo text visible.

**Comment by xxyzz** [comment]
The copy icon and the copied message are both invisible, but I can't find a way to fix this.

File: src/config/dynamic-theme-fixes.config (hunk 1 above)
Comment (Gusted) [reply src/config/dynamic-theme-fixes.config]: Hmm, span.icon doesn't seem to work for me, just `.logo-text`...
image

**Review by Gusted — APPROVED** [review]

**Comment by Gusted** [comment]
> The copy icon and the copied message are both invisible, but I can't find a way to fix this

`button.copy`?
Copied message seems to be correct already?

**Comment by xxyzz** [comment]
This copy icon and a "Copied" message pop up when you click it: https://developer.mozilla.org/en-US/docs/Learn/CSS/Building_blocks/Selectors#type_class_and_id_selectors
Screen Shot 2022-03-04 at 19 14 05

File: src/config/dynamic-theme-fixes.config (hunk 1 above)
Comment (xxyzz) [reply src/config/dynamic-theme-fixes.config]: I can't make the logo text both in https://developer.mozilla.org/en-US/ and https://developer.mozilla.org/en-US/docs/Learn/CSS/Building_blocks/Selectors white... Darkreader throw an error:

<details>
Content Security Policy: The page’s settings blocked the loading of a resource at inline (“script-src”). Source: (function injectProxy(enableStyleSheetsProxy) {
        document.dispatchEvent(new CustomEvent('__darkreader__inlineScriptsAllowed'));
        const addRuleDescriptor = Object.getOwnPropertyDescriptor(CSSStyleSheet.prototype, 'addRule');
        const insertRuleDescriptor = Object.getOwnPropertyDescriptor(CSSStyleSheet.prototype, 'insertRule');
        const deleteRuleDescriptor = Object.getOwnPropertyDescriptor(CSSStyleSheet.prototype, 'deleteRule');
        const removeRuleDescriptor = Object.getOwnPropertyDescriptor(CSSStyleSheet.prototype, 'removeRule');
        const documentStyleSheetsDescriptor = enableStyleSheetsProxy ?
            Object.getOwnPropertyDescriptor(Document.prototype, 'styleSheets') : null;
        const shouldWrapHTMLElement = location.hostname.endsWith('baidu.com');
        const getElementsByTagNameDescriptor = shouldWrapHTMLElement ?
            Object.getOwnPropertyDescriptor(Element.prototype, 'getElementsByTagName') : null;
        const cleanUp = () => {
            Object.defineProperty(CSSStyleSheet.prototype, 'addRule', addRuleDescriptor);
            Object.defineProperty(CSSStyleSheet.prototype, 'insertRule', insertRuleDescriptor);
            Object.defineProperty(CSSStyleSheet.prototype, 'deleteRule', deleteRuleDescriptor);
            Object.defineProperty(CSSStyleSheet.prototype, 'removeRule', removeRuleDescriptor);
            document.removeEventListener('__darkreader__cleanUp', cleanUp);
            document.removeEventListener('__darkreader__addUndefinedResolver', addUndefinedResolver);
            if (enableStyleSheetsProxy) {
                Object.defineProperty(Document.prototype, 'styleSheets', documentStyleSheetsDescriptor);
            }
            if (shouldWrapHTMLElement) {
                Object.defineProperty(Element.prototype, 'getElementsByTagName', getElementsByTagNameDescriptor);
            }
        };
        const addUndefinedResolver = (e) => {
            customElements.whenDefined(e.detail.tag).then(() => {
                document.dispatchEvent(new CustomEvent('__darkreader__isDefined', { detail: { tag: e.detail.tag } }));
            });
        };
        document.addEventListener('__darkreader__cleanUp', cleanUp);
        document.addEventListener('__darkreader__addUndefinedResolver', addUndefinedResolver);
        const updateSheetEvent = new Event('__darkreader__updateSheet');
        function proxyAddRule(selector, style, index) {
            addRuleDescriptor.value.call(this, selector, style, index);
            if (this.ownerNode && !this.ownerNode.classList.contains('darkreader')) {
                this.ownerNode.dispatchEvent(updateSheetEvent);
            }
            return -1;
        }
        function proxyInsertRule(rule, index) {
            const returnValue = insertRuleDescriptor.value.call(this, rule, index);
            if (this.ownerNode && !this.ownerNode.classList.contains('darkreader')) {
                this.ownerNode.dispatchEvent(updateSheetEvent);
            }
            return returnValue;
        }
        function proxyDeleteRule(index) {
            deleteRuleDescriptor.value.call(this, index);
            if (this.ownerNode && !this.ownerNode.classList.contains('darkreader')) {
                this.ownerNode.dispatchEvent(updateSheetEvent);
            }
        }
        function proxyRemoveRule(index) {
            removeRuleDescriptor.value.call(this, index);
            if (this.ownerNode && !this.ownerNode.classList.contains('darkreader')) {
                this.ownerNode.dispatchEvent(updateSheetEvent);
            }
        }
        function proxyDocumentStyleSheets() {
            const getCurrentValue = () => {
                const docSheets = documentStyleSheetsDescriptor.get.call(this);
                return Object.setPrototypeOf([...docSheets].filter((styleSheet) => {
                    return !styleSheet.ownerNode.classList.contains('darkreader');
                }), StyleSheetList.prototype);
            };
            let elements = getCurrentValue();
            const styleSheetListBehavior = {
                get: function (_, property) {
                    return getCurrentValue()[property];
                }
            };
            elements = new Proxy(elements, styleSheetListBehavior);
            return elements;
        }
        function proxyGetElementsByTagName(tagName) {
            if (tagName !== 'style') {
                return getElementsByTagNameDescriptor.value.call(this, tagName);
            }
            const getCurrentElementValue = () => {
                const elements = getElementsByTagNameDescriptor.value.call(this, tagName);
                return Object.setPrototypeOf([...elements].filter((element) => {
                    return !element.classList.contains('darkreader');
                }), NodeList.prototype);
            };
            let elements = getCurrentElementValue();
            const nodeListBehavior = {
                get: function (_, property) {
                    return getCurrentElementValue()[Number(property)];
                }
            };
            elements = new Proxy(elements, nodeListBehavior);
            return elements;
        }
        Object.defineProperty(CSSStyleSheet.prototype, 'addRule', Object.assign({}, addRuleDescriptor, { value: proxyAddRule }));
        Object.defineProperty(CSSStyleSheet.prototype, 'insertRule', Object.assign({}, insertRuleDescriptor, { value: proxyInsertRule }));
        Object.defineProperty(CSSStyleSheet.prototype, 'deleteRule', Object.assign({}, deleteRuleDescriptor, { value: proxyDeleteRule }));
        Object.defineProperty(CSSStyleSheet.prototype, 'removeRule', Object.assign({}, removeRuleDescriptor, { value: proxyRemoveRule }));
        if (enableStyleSheetsProxy) {
            Object.defineProperty(Document.prototype, 'styleSheets', Object.assign({}, documentStyleSheetsDescriptor, { get: proxyDocumentStyleSheets }));
        }
        if (shouldWrapHTMLElement) {
            Object.defineProperty(Element.prototype, 'getElementsByTagName', Object.assign({}, getElementsByTagNameDescriptor, { value: proxyGetElementsByTagName }));
        }
    })(!undefined).
</details>

Comment (Gusted) [reply src/config/dynamic-theme-fixes.config]: That error is unrelated.

**Review by Gusted — APPROVED** [review]

**Pull request merged**